draft-scenarios/coops/close_to_enemies.tex

```diff
@@ -0,0 +1,222 @@
+      \item Second wave of enemy clan armies attacks the towns. (see below)
+      \item Each player who defended his own town, search (3) the \textit{Artifact Deck}.
+     \end{itemize}
+   \item \textbf{\nth{6} Round:}
+    \begin{itemize}
+      \item Third wave of enemy clan armies attacks the towns. (see below)
+      \item Each player who defended his own town, search (3) the \textit{Artifact Deck}.
+    \end{itemize}
+   \item \textbf{\nth{7} Round:}
+    \begin{itemize}
+      \item During this Round, Players can discard and draw Cards at the start of their turn twice rather than once.
+    \end{itemize}
+  \end{itemize}
+
+\subsection*{\MakeUppercase{Enemy clan waves}}
+\begin{itemize}
+  \item Unconquered enemy Settlements will send clan armies to attack your Town!
+  \item A battle occurs at the start of each player's turn.
+  \item Before combat, you may use your Building, Spell Book, and Population Tokens, as well as Might and Magic Cards. However, you cannot move your Hero.
+  \item Each player must defend their own Town.
+  \item If you have a Citadel, the \textit{Arrow Tower} attack is reduced. The attacks for first/second/third wave become 2/3/4 \svg{attack}.
+  \item This combat does not cost movement points.
+  \item Refer to the \textit{Enemy Clan Armies} table for siege army strength.
+  \item If your main Hero is present in any of allied town (not settlement) and Clan wave attack your allied town where is no main Hero, you can participate on defending allied castle by using Might and Magic Cards (Simply: you defend your teammate's castle with their Units and your Hero's Cards).
+  \item Gain one \svg{experience} when successfully defending your Town, regardless of whether your main Hero participated.
+\end{itemize}
+
+\subsection*{\MakeUppercase{Settlements defenders}}
+\begin{itemize}
+  \item The strength of the army in Settlement is determined by the table \textit{Enemy's Clans armies} (last row).
+  \item After draw an enemy army, Player can discard one enemy Unit Card and draw a new Unit of same Tier.
+  \item The battle does not cost any extra movement points.
+  \item In this battles, you has to use AI Cards.
+  \item After won battle, you gain one Level.
+\end{itemize}
+
+\columnbreak
+
+\vspace*{\fill}
+
+\end{multicols}
+
+\vspace{2em}
+
+\begin{center}
+  \hommtable[]{16}{
+    \centering
+    \medskip
+    \textbf{Enemy's Clans armies}\\
+
+    \newcommand{\bronze}[0]{\svgunit[12]{bronze}}
+    \newcommand{\silver}[0]{\svgunit[12]{silver}}
+    \newcommand{\golden}[0]{\svgunit[12]{golden}}
+    \newcommand{\azure}[0]{\svgunit[12]{azure}}
+
+      \begin{tabularx}{\linewidth}{p{0.2\linewidth}XXXX} &
+      \darkcell{\raisebox{-.2\height}{\svgunit[20]{bronze}} \textbf{Easy}} &
+      \darkcell{\raisebox{-.2\height}{\svgunit[20]{silver}} \textbf{Normal} &
+      \darkcell{\raisebox{-.2\height}{\svgunit[20]{golden}} \textbf{Hard}} &
+      \darkcell{\raisebox{-.2\height}{\svgunit[20]{azure}} \textbf{Impossible}}\\
```

Comment:
```suggestion
      \darkcell{\textbf{Easy}} &
      \darkcell{\textbf{Normal}} &
      \darkcell{\textbf{Hard}} &
      \darkcell{\textbf{Impossible}}\\
```
Sorry, a syntax error in my suggestion broke the build. On 2nd thought, though, I think it's better without those unit stars in the header row.

![image](https://github.com/user-attachments/assets/26789ed3-be8c-4e64-b1c3-3122da50cad8)